### DISCUSSION TITLE
Installation problem

### DISCUSSION THREAD

**Discussion by ShakhmatovIS** [PMEAL/OpenPNM#2584] [Q&A] [answered]
Colleagues, I have a problem. Installed according to the instructions.
I'm trying to run

`import openpnm as op
import matplotlib.pyplot as plt

pn = op.network.Cubic(shape=[5, 5, 5], spacing=[1, 1, 1])
pn.Np`

And I get an error
CRITICAL   : front and back labels have been switched to obey the right-hand rule 
SOURCE     : openpnm.network.Cubic.__init__ 
TIME STAMP : 2022-09-30 08:10:45,916

I tried reinstalling everything, it doesn't work out

**Comment by jgostick** [comment]
This is fine...we over-reacted when we labelled this logger message as 'critical'...probably just a simple 'warning' would have been more appropriate.  We just didn't want people to start applying BCs to Front/Back pores and getting inconsistent results.

**Comment by ShakhmatovIS** [comment, marked as answer]
But at the moment I can't count anything at all from the examples. Perhaps there is some solution. I really wanted to use your module for my research work

**Reply by ShakhmatovIS** [reply]
The problem was solved by switching to an older version of Ubuntu 20.04.
